src/frontend/components/property-type/array/edit.tsx

```diff
@@ -20,14 +20,12 @@ type ItemRendererProps = {
 }
 
 const ItemRenderer: React.FC<EditProps & ItemRendererProps> = (props) => {
-  const { ItemComponent, property, onDelete, index, record, isDraggable } = props
-  const uniqueDraggableId = window.btoa(unescape(encodeURIComponent(`${JSON.stringify(flat.get(record.params, property.path))}-${property.path}`)))
+  const { ItemComponent, property, onDelete, index, isDraggable } = props
 
   return (
     <Draggable
-      draggableId={uniqueDraggableId}
+      draggableId={property.path}
```

Comment:
Does this fix the issue? `property.path` is still not unique as it changes when you drag an element between indices.

Reply:
That's a fair point will have to dive into that.

Reply:
There doesn't seem to be a convenient value to use as a unique key.

Reply:
One solution I thought of would be to create a "virtual" property `draggableId` for all existing array elements when the HOC component first loads, then assign it also when you add new elements. We could use some kind of library like `nanoid` to create a unique identifier. There could be an issue where AdminJS also sends this property in request payload and an adapter tries to save it. AFAIK some ORMs skip attributes that are not defined in models but other ones could throw errors. A solution to this could be that we treat all properties prefixed with double underscore (`__typename`, `__draggableId`, etc) as virtual/readonly and remove them when form is submitted.

Reply:
Would you want that solution as part of this PR?

Reply:
Ideally, yes. If you have doubts though and `property.path` change works exactly the same as `uniqueDraggableId` (as it was before) then it's fine to do it another time.

Reply:
I would prefer handling it in a separate PR, the current implementation has some quirks like focus going to the wrong field when dragging elements. But that is an existing issue.